Library/Homebrew/exceptions.rb

```diff
@@ -65,6 +65,22 @@ def to_s; <<-EOS.undent
   end
 end
 
+class TapFormulaAmbiguityError < RuntimeError
+  attr_reader :name, :paths
+
+  def initialize name, paths
+    @name = name
+    @paths = paths
+  end
+
+  def to_s; <<-EOS.undent
```

Comment:
Instead of redefining `to_s`, just do `super <<-EOS.undent ...` inside initialize